tests/examples/shims/root.lua

```diff
@@ -1,9 +1,126 @@
 local root = {_tags={}}
 
+local hotkeys = nil
+
 function root:tags()
     return root._tags
 end
 
+function root:size() --TODO use the screens
+    return 0, 0
+end
+
+function root:size_mm()
+    return 0, 0
+end
+
+function root.cursor() end
+
+-- GLOBAL KEYBINDINGS --
+
+local keys = {}
+
+-- Note: the tests are not expected to create 1000 of them, so to keep the
+-- implementation minimal, it doesn't try to hash them and simply loop over
+-- every entry.
```

Comment:
Uhm, the C code does not hash them either. It just does a linear search for matching keys. So, I do not see the point of this comment and would just remove it.